tree.go

```diff
@@ -313,14 +316,6 @@ func (n *InternalNode) SetChild(i int, c VerkleNode) error {
 	return nil
 }
 
-// TouchCoW is a helper function that will mark a child as
-// "inserted into". It is used by the conversion code to
-// mark reconstructed subtrees as 'written to', so that its
-// root commitment can be computed.
-func (n *InternalNode) TouchCoW(index byte) {
-	n.cowChild(index)
-}
```

Comment:
This wasn't used, so taking the liberty to remove it. I suspect was used in some early approaches of the full tree conversion. I'd prefer to remove this since feels too "hacky" to be a public API.

Reply:
@gballet, since we're recovering the `MergesTrees(...)` that uses this method, I'm also recovering this one but at least leave it unexported.